frontend/src/js/button/DownloadButton.tsx

```diff
@@ -13,8 +15,31 @@ const Link = styled("a")`
   line-height: 1;
 `;
 
+const fileTypeToIcon: Record<string, IconName> = {
+  ZIP: "file-archive",
+  XLSX: "file-excel",
+  PDF: "file-pdf",
+  CSV: "file-csv",
+};
+function getFileIcon(label: string): IconName {
+  // Editor Requests
+  if (label in fileTypeToIcon) {
+    return fileTypeToIcon[label];
+  }
+
+  // Forms
+  if (label.includes(".")) {
+    const ext = label.split(".").pop()?.toLocaleUpperCase();
```

Comment:
What's happening here? This is getting an ending, right? Can we combine this with [getEnding](https://github.com/ingef/conquery/pull/2931/files#diff-085890ab95def8ca4d7754efd73abd80ab7b62fa279937ecd1ab2d511db5875eR52) here?